docs-web/app-protect/troubleshooting.md

```diff
@@ -0,0 +1,95 @@
+
+### Checking the Events of an Ingress Resource
+
+Follow the steps of [Checking the Events of an Ingress Resource](/troubleshooting/#checking-the-events-of-an-ingress-resource).
+
+### Checking events of APLogConf
+
+After you create or update an APLogConf, you can immediately check if the NGINX configuration was successfully applied by NGINX:
+```
+$ kubectl describe aplogconf logconf
+Name:         logconf
+Namespace:    default
+. . . 
+Events:
+  Type    Reason          Age   From                      Message
+  ----    ------          ----  ----                      -------
+  Normal  AddedOrUpdated  11s   nginx-ingress-controller  AppProtectLogConfig  default/logconf was added or updated
+```
+Note that in the events section, we have a `Normal` event with the `AddedOrUpdated` reason, which informs us that the configuration was successfully applied.
+
+### Checking events of APPolicy
+
+After you create or update an APPolicy, you can immediately check if the NGINX configuration was successfully applied by NGINX:
+```
+$ kubectl describe appolicy dataguard-alarm
+Name:         dataguard-alarm
+Namespace:    default
+. . . 
+Events:
+  Type    Reason          Age    From                      Message
+  ----    ------          ----   ----                      -------
+  Normal  AddedOrUpdated  2m25s  nginx-ingress-controller  AppProtectPolicy default/dataguard-alarm was added or updated
+```
+Note that in the events section, we have a `Normal` event with the `AddedOrUpdated` reason, which informs us that the configuration was successfully applied.
+ 
+## Running App Protect in Debug Mode
+
+Running the Ingress Controller in debug mode will also run the App Protect module in debug mode. Follow the steps of [Running NGINX in the Debug Mode](/nginx-ingress-controller/troubleshooting/#running-nginx-in-the-debug-mode).
+
+## Known Issues
+
+When using the Ingress Controller with the App Protect module, the following issues might happen depending on the number of Ingress Resources with App Protect enabled in your cluster:
+
+### NGINX reload times increase
+
+The Ingress Controller reloads NGINX whenever there is a change that requires NGINX to reload to apply a new configuration. Without the App Protect module enabled, usual reload times are around 150ms. If App Protect module is enabled and is being used by any number of Ingress Resources, these reloads might take a few seconds instead. If you are running more than one instance of the Ingress Controller, because there is no order on how the Ingress Controller processes the Kubernete Reources, it might occur that different instances have different NGINX configurations until all of them have finished with the reloads.
```

Comment:
```suggestion
When you make a change that requires NGINX to apply a new configuration, the Ingress Controller reloads NGINX automatically. Without the App Protect module enabled, usual reload times are around 150ms. If App Protect module is enabled and is being used by any number of Ingress Resources, these reloads might take a few seconds instead. 

> **Note**: If you are running more than one instance of the Ingress Controller, the extended reload time may cause the NGINX configuration of your instances to be out of sync. This can occur because there is no order imposed on how the Ingress Controller processes the Kubernetes Resources. The configurations will be the same after all instances have completed the reload.
```